.github/CODEOWNERS

```diff
@@ -16,8 +16,8 @@
 /apps/web*/                     	@island-is/kosmos-kaos @island-is/aranja
```

Comment:
Are all these indentation changes intended?

Reply:
Yes, they were. The file was kind of messy in that regard when I was adding my line to it. I noticed later that the file contains a mixture of tabs and spaces. Do we care?

Reply:
Ah I see it now. Yes, can you have it consistent - I guess we're using spaces in other places, can you replace all tabs with spaces while you're in there? :)

Reply:
Maybe there should be some auto formatting of this file.

Reply:
Agreed. https://github.com/island-is/island.is/issues/1259

Reply:
Done replacing tabs.